### PR TITLE
chore: Update Azure CNI to v1.2.7

### PR DESCRIPTION
**Pull request by AbelHu**
Update Azure CNI to [v1.2.7](https://github.com/Azure/azure-container-networking/releases/tag/v1.2.7)

### CODE REVIEW

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -148,10 +148,9 @@ function Get-FilesToCacheOnVHD {
             "https://acs-mirror.azureedge.net/kubernetes/v1.20.2/windowszip/v1.20.2-1int.zip"
         );
         "c:\akse-cache\win-vnet-cni\" = @(
-            "https://acs-mirror.azureedge.net/azure-cni/v1.2.0/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.0.zip",
-            "https://acs-mirror.azureedge.net/azure-cni/v1.2.0_hotfix/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.0_hotfix.zip",
             "https://acs-mirror.azureedge.net/azure-cni/v1.2.2/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.2.zip",
-            "https://acs-mirror.azureedge.net/azure-cni/v1.2.6/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.6.zip"
+            "https://acs-mirror.azureedge.net/azure-cni/v1.2.6/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.6.zip",
```
Comment (mainred) [inline vhdbuilder/packer/configure-windows-vhd.ps1:152]: How do we assure 1.2.0 is not used for existing clusters and should we also remove Linux 1.2.0 and 1.2.0_hotfix cache?

Comment (AbelHu) [reply vhdbuilder/packer/configure-windows-vhd.ps1:152]: Removing cached file does not break functionality but only increase the provisioning time to download cni from the internet. 
In the past, we release the new VHD with the new CNI version change together. Now we are using toggle to update Azure CNI version. They cannot always match. Sometimes cni version is changed before a new VHD is built.
I do not want to remove Linux cached files and leave them to Linux guys to remove it. Linux is still using 1.2.0_hotfix.

**Review by mainred — APPROVED** [review]
Looks good from Windows side

**Comment by paulgmiller** [comment]
There's a swift azure cni right below the linux one we should update with 1.2.7 as well.

File: vhdbuilder/packer/configure-windows-vhd.ps1 (hunk 1 above)
Comment (paulgmiller) [reply vhdbuilder/packer/configure-windows-vhd.ps1:152]: We keep 3 traditionally if we need to roll back toggles.  so I'm fine with this. Good to check the toggle isn't behind though.

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -105,6 +105,7 @@ cat << EOF >> ${VHD_LOGS_FILEPATH}
 EOF
 
 VNET_CNI_VERSIONS="
+1.2.7
 1.2.6
 1.2.0_hotfix
 1.2.0
```
Comment (paulgmiller) [inline vhdbuilder/packer/install-dependencies.sh:111]: Can remove this

Comment (paulgmiller) [reply vhdbuilder/packer/install-dependencies.sh:111]: Also update SWIFT_CNI_VERSIONS below? once the versions converge I'll combine them.

**Review by paulgmiller — APPROVED** [review]
Just two comments. If you fix those I can merge those tonight.

File: vhdbuilder/packer/test/linux-vhd-content-test.sh [hunk 3]
```diff
@@ -116,7 +116,7 @@ filesToDownload='
   "fileName":"azure-vnet-cni-linux-amd64-v*.tgz",
   "downloadLocation":"/opt/cni/downloads",
   "downloadURL":"https://acs-mirror.azureedge.net/azure-cni/v*/binaries",
-  "versions":["1.2.6", "1.2.0_hotfix","1.2.0"]
+  "versions":["1.2.7", "1.2.6", "1.2.0_hotfix","1.2.0"]
```
Comment (paulgmiller) [inline vhdbuilder/packer/test/linux-vhd-content-test.sh:119]: update swift ut below too

File: vhdbuilder/packer/install-dependencies.sh (hunk 2 above)
Comment (AbelHu) [reply vhdbuilder/packer/install-dependencies.sh:111]: @paulgmiller Do you mean 1.2.0 or 1.2.6 or both?

Comment (AbelHu) [reply vhdbuilder/packer/install-dependencies.sh]: Removed Linux cni 1.2.0 and added swift cni 1.2.7

Comment (paulgmiller) [reply vhdbuilder/packer/install-dependencies.sh]: Thanks that's what I meant

**Pull request merged**